dinky-admin/src/main/java/org/dinky/controller/TaskController.java

```diff
@@ -168,8 +171,8 @@ public Result<ObjectNode> getJobPlan(@ProcessId @RequestBody TaskDTO taskDTO) {
             dataType = "Task",
             paramType = "body",
             dataTypeClass = Task.class)
```

Comment:
change dataTypeClass to TaskSaveDTO.class

Reply:
ok